SAT/Preprocess.cpp

```diff
@@ -245,6 +250,7 @@ void Preprocess::propagateUnits(SATClauseIterator clauses,
     unsigned newLen=clen-removedLitIndexes.length();
 
     if(newLen==1) {
```

Comment:
The static analyser can find a lengthy code path where `kept` isn't initialised here. I think in practice it's OK, but this serves as documentation and also prevents a disaster if I'm wrong.